composer.json

```diff
@@ -23,8 +23,8 @@
     "require": {
         "php": "^7.1",
         "psr/log": "^1.0",
-        "sonata-project/block-bundle": "^3.11",
-        "sonata-project/core-bundle": "^3.9",
+        "sonata-project/block-bundle": "^4.0",
```

Comment:
We should stop building with PHP < 7.3 based on this constraint ([ref](https://travis-ci.org/sonata-project/SonataFormatterBundle/jobs/622760538#L382)).